tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -3990,4 +3990,227 @@ public function provide_test_is_valid_optimized_checkout_page() {
+				],
+			]
+		);
+		WC_Stripe_Database_Cache::set( 'checkout_session_' . $checkout_session_id, $checkout_session );
+
+		ob_start();
+		$this->mock_gateway->add_currency_conversion_notice( $order );
+		$output = ob_get_clean();
+
+		WC_Stripe_Database_Cache::delete( 'checkout_session_' . $checkout_session_id );
+
+		$expected_amount     = WC_Stripe_Helper::get_woocommerce_amount_from_stripe_amount( 1500, 'eur' );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\WC_Stripe_Helper' in method 'test_add_currency_conversion_notice_outputs_notice_with_converted_amount_and_rate'. (undefined)

(StaticAccess)